eng/pipelines/templates/jobs/archetype-sdk-client.yml

```
@@ -84,6 +87,20 @@ jobs:
 
           - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
 
+          - task:  UniversalPackages@0
+            displayName: Download ApiView Parser.
+            inputs:
+              command: download
+              vstsFeed: 'internal/AzureSDKForCpp'
+              vstsFeedPackage: 'parseazuresdkcpp'
+              downloadDirectory: '$(System.DefaultWorkingDirectory)/parser'
+              vstsPackageVersion: 0.6.0
+            condition: >-
+                and(
+                  succeeded(),
+                  ne(variables['Skip.CreateApiReview'], 'true'),eq(variables['System.TeamProject'], 'internal')
```

Totally optional nit: 

```suggestion
                  ne(variables['Skip.CreateApiReview'], 'true'),
                  eq(variables['System.TeamProject'], 'internal')
```